tests/chainer_tests/functions_tests/loss_tests/test_softmax_cross_entropy.py

```diff
@@ -84,44 +94,58 @@ def check_forward(self, x_data, t_data, use_cudnn=True):
         testing.assert_allclose(
             loss_expect, loss_value, **self.check_forward_options)
 
+    @testing.with_requires('numpy>=1.10')
```

Comment:
Because numpy.broadcast_to is available only from numpy>=1.10. I will add a comment before the decorator.